raaghu-components/src/rds-comp-apply-for-position/rds-comp-apply-for-position.tsx

```
@@ -228,6 +228,7 @@ const isPeriodValid = (period: any) => {
                 }}
                 value={formData?.coverLetter}
                 dataTestId="cover-letter"
+                showTitle={true}
```

ESLint

[error] Expected indentation of 32 spaces but found 16.

(indent)